tests/scale_test/notifier.py

```diff
@@ -201,13 +205,13 @@ def batch(ctx, bundle_list_file, run_mode):
 
     if run_mode == 'async':
         logging.info('Sending notifications asynchronously...\n')
-        async_notify(bundles, lira_url, label, es_query_path, auth_token, save_path)
+        async_notify(bundles, lira_url, label, es_query_path, auth_dict, save_path)
     elif run_mode == 'sync':
         logging.info('Sending notifications synchronously...\n')
-        linear_notify(bundles, lira_url, label, es_query_path, auth_token, save_path)
+        linear_notify(bundles, lira_url, label, es_query_path, auth_dict, save_path)
 
```

Comment:
Is there a fall through state? What happens if the boolean blocks above are both false. Is that expected? Should an `else` block throw an error or maybe it defaults to sync? You can make a decision.

Reply:
Thanks for bringing up this, this is a good point.

Specifically for this case, the mutual exclusiveness and non-null property between `{async, sync}` is ensured by the `click` lib, so if users don't specify anything, the default value of `run_mode` would be `async`. To be more elegant, we could have used something like:

```python
    if run_mode == 'async':
        async_notify(bundles, lira_url, label, es_query_path, auth_dict, save_path)
    else:
        linear_notify(bundles, lira_url, label, es_query_path, auth_dict, save_path)
```
instead of an open `if...elif...` without `else` whereas I think it's kinda making the code implicit in terms of "readability". Now by revisiting it, I'd prefer using 2 separate `if`s here.